cypress/integration/group2/searchTable.ts

```diff
@@ -69,3 +69,30 @@ describe('Dockstore tool/workflow search table', () => {
+    });
+    cy.visit('/search');
+    cy.get('#mat-select-0 ')
+      .click()
+    cy.get('#mat-option-1 ')
+      .click()
+    cy.get('.cdk-column-name > a ').contains('garyluu/dockstore-cgpmap/cgpmap-cramOut').should('exist')
```

Comment:
not sure how "real" this data is, but ideally this would reference a dockstore-testing workflow, something in one of our organizationally controlled repos

Reply:
This is for the cypress tests which doesn't have elastic search running so database loaded into the cypress test only has three workflows and this is the first one.

Reply:
per gary's suggestion, I put the response I re-used in a fixture and made both tests use the fixture